frontstage/templates/account/account-change-password.html

```diff
@@ -0,0 +1,162 @@
+                        "url": "#" ~ error ~ '_error',
+                        "classes": "js-inpagelink"
+                    }
+                ) %}
+            {% endfor %}
+        {% endif %}
+
+        {% call
+            onsPanel({
+                "type": "error",
+                "classes": "u-mb-s",
+                "title":  errorTitle
+            })
+        %}
+            <p>This <strong>must be corrected</strong> to continue.</p>
+
+            {% from "components/lists/_macro.njk" import onsList %}
+            {{
+                onsList({
+                    "classes": "list--bare",
+                    "itemsList": errorData
+                })
+            }}
+        {% endcall %}
+    {% endif %}
+<h1 class="u-fs-xl">Change your password</h1>
+<form action="" method="post">
+        {% if errorType == 'failed' %}
+            {% set errorPassword = { "text": "Incorrect current password", "id": 'password_error' } %}
+        {% elif errors.password %}
+                {% set errorPassword = { "text": errors['password'][0],  "id": 'password_error' } %}
+        {% endif %}
+        {{ form.csrf_token }}
+            {{
+                onsInput({
+                    "id": "password",
+                    "name": "password",
+                    "type": "text",
+                    "label": {
+                        "text": 'Enter your current password'
+                    },
+                    "value": password,
+                    "error": errorPassword,
+                })
+            }}
+            <div class="field">
+                <h2 class="u-mt-l">Create a new password</h2>
+                <div class="message">
+                    <p>Your password must have:</p>
+                    <ul>
+                        <li>at least 8 characters </li>
+                        <li>at least 1 uppercase letter</li>
+                        <li>at least 1 symbol (eg: ?!£%) </li>
+                        <li>at least 1 number</li>
+                    </ul>
+                </div>
+                <div>
+                {%- call onsFieldset({
+                        "legend": "New password",
+                        "legendClasses": "u-vh"
+                    }) -%}
+                {% if errors.new_password %}
+                    {% set newPasswordError = { "text": errors['new_password'][0],  "id": 'new_password_error' } %}
+                {% endif %}
+                {{
+                    onsPassword({
+                        "id": "new_password",
+                        "name": "new_password",
+                        "type": "text",
+                        "label": {
+                            "text": "New Password"
+                        },
+                        "showPasswordText": "Show password",
+                        "error": newPasswordError
+                    })
+                }}
+                </div>
+            </div>
+            {{
+                onsInput({
```

Comment:
This needs to be onsPassword.

It's not in the design, but I think this retype password field is missing a 'show password' checkbox.  The registration screen has 2 so theres no reason why this shouldn't have 2.  I've asked Jeremy about this.

Reply:
I did it because of the prototype. Will change it :-(